src/test/run-pass/union/union-pat-refutability.rs

```diff
@@ -29,7 +29,7 @@ fn is_zero(v: Value) -> bool {
     unsafe {
         match v {
             Value { tag: Tag::I, u: U { i: 0 } } => true,
-            Value { tag: Tag::F, u: U { f: 0.0 } } => true,
+            Value { tag: Tag::F, u: U { u: 0 } } => true,
```

Comment:
Although i guess one could argue that people should just match on the bit pattern (i.e. The integer variant)